src/app/component/Klarna/Klarna.component.js

```diff
@@ -80,14 +80,67 @@ export class Klarna extends PureComponent {
             script.parentNode.removeChild(script);
         }
 
+        const klarnaScript = document.createElement('script');
+        klarnaScript.setAttribute('id', KLARNA_SCRIPT_ID);
+        klarnaScript.setAttribute('src', 'https://x.klarnacdn.net/kp/lib/v1/api.js');
+        klarnaScript.async = true;
+        document.head.appendChild(klarnaScript);
```

Comment:
Does not this break if going to and from checkout multiple times? The script should be removed from head on un-mount, no?

Reply:
This assume script removes it self, but the problem is that when it is called, it also triggers another script ( klarna-payments-device-recognition ) that is not removing it self. So Just added logic to remove it if payment was shown. Now if we go back then it also removed.

Reply:
> Also, is this expected, that the composer.json has no Kalrna module in it? If yes, then alright :D

I think no, without it klarna won't work :'D. So I added last version into composer